config/kafkaconnect/config.go

```diff
@@ -0,0 +1,57 @@
+package kafkaconnect
+
+import (
+	"time"
+
+	"github.com/crossplane/upjet/pkg/config"
+	"github.com/upbound/provider-aws/config/common"
+)
+
+// Configure adds configurations for the kafkaconnect group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_mskconnect_connector", func(r *config.Resource) {
+		// This will always refer to a Cluster in the kafka api group, if it refers to any managed resource at all,
+		// but which property from the status of that cluster to use depends on the authentication mechanism chosen.
+		delete(r.References, "kafka_cluster.apache_kafka_cluster.bootstrap_servers")
+		r.References["kafka_cluster.apache_kafka_cluster.vpc.security_groups"] = config.Reference{
+			TerraformName:     "aws_security_group",
+			SelectorFieldName: "SecurityGroupSelector",
+			RefFieldName:      "SecurityGroupRefs",
+		}
+		r.References["kafka_cluster.apache_kafka_cluster.vpc.subnets"] = config.Reference{
+			TerraformName:     "aws_subnet",
+			SelectorFieldName: "SubnetSelector",
+			RefFieldName:      "SubnetRefs",
+		}
+		r.References["log_delivery.worker_log_delivery.s3.bucket"] = config.Reference{
+			TerraformName: "aws_s3_bucket",
+		}
+		r.References["log_delivery.worker_log_delivery.cloudwatch_logs.log_group"] = config.Reference{
+			TerraformName: "aws_cloudwatch_log_group",
+		}
+		r.References["log_delivery.worker_log_delivery.firehose.delivery_stream"] = config.Reference{
+			TerraformName: "aws_kinesis_firehose_delivery_stream",
+			Extractor:     `github.com/crossplane/upjet/pkg/resource.ExtractParamPath("name",true)`,
+		}
+		r.References["worker_configuration.arn"] = config.Reference{
+			TerraformName: "aws_mskconnect_worker_configuration",
+			Extractor:     common.PathARNExtractor,
+		}
+		r.References["plugin.custom_plugin.arn"] = config.Reference{
```

Comment:
Since this reference is clearly in the terraform example, it should come automatically. Is there a reason to add it again?

Reply:
Not a strong one, just my preference for being explicit over implicit. I wasn't looking at the examples when I was writing the config, I was just specifying all the references based on my understanding of the schema. 

My preference would be to leave this in, but if you'd rather I remove it, I will.

Reply:
We generally prefer not to define automatically defined references. I don't have a strong reason for it to be removed here either, so I leave the choice up to you.